modules/Kconfig.mcuboot

```diff
@@ -41,11 +41,12 @@ config NORDIC_QSPI_NOR_FLASH_LAYOUT_PAGE_SIZE
 config BOOT_IMAGE_ACCESS_HOOKS_FILE
 	string
 	default "${ZEPHYR_NRF_MODULE_DIR}/modules/mcuboot/hooks/nrf53_hooks.c"
-	depends on MCUBOOT
+	depends on BOOT_IMAGE_ACCESS_HOOKS
 
 config BOOT_IMAGE_ACCESS_HOOKS
 	bool
 	default y if UPDATEABLE_IMAGE_NUMBER > 1 && SOC_NRF5340_CPUAPP && PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY
+	depends on MCUBOOT
```

Comment:
nit: Are we sure that this is needed? I was my understanding that this file was sourced only by the MCUboot child image, which implies that `MCUBOOT` is always set.

Reply:
no, this file is ALWAYS sourced:
https://github.com/nrfconnect/sdk-nrf/blob/3e1f601eb6c13d9a6189505fab0bc7ee1e4670fe/modules/Kconfig#L7

Reply:
Thanks for clarifying